t/helper/test-quote.c

```diff
@@ -0,0 +1,30 @@
+#include "test-tool.h"
+#include "quote.h"
+#include "strbuf.h"
+#include "string.h"
+
+int cmd__quote_buf_pretty(int argc, const char **argv)
+{
+	struct strbuf buf_payload = STRBUF_INIT;
+	char* nullString = 0;
```

Comment:
you can spell this as

    char *nullString = NULL;

also, the '*' should usually touch the variable not the base type.  (nit nit nit :-)